torch/ao/quantization/_pt2e/quantizer/quantizer.py

```diff
@@ -98,6 +103,29 @@ class QuantizationConfig:
     ],
 )
 
+@dataclass
+class QuantizationAnnotation:
```

Comment:
If we want to add more fields only for customized backend recipe. Take IPEX recipes for example, should we consider a `IPEXQuantizationAnnotation` class which inherit this class with added fields?

Reply:
I think it's better for you to define a class separately and attach it to meta field of node, e.g.:

```
node.meta["ipex_annotation"] = IPEXAnnotation(...)
```

Reply:
But looks like it's hardcoded checkout of filed `quantization_annotation` in node.meta  https://github.com/pytorch/pytorch/blob/433672ec9cc4fe11bfb1e53b35601b76316899d4/torch/ao/quantization/fx/prepare.py#L527

Reply:
it will be preserved and should not interact with prepare